pkg/sysregistriesv2/system_registries_v2.go

```
@@ -306,6 +318,9 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 		}
 
 		if reg.Prefix == "" {
+			if reg.Location == "" {
+				return err
```

`err` is necessarily `nil` at this point, this should return a helpful error message.

---

ack

---

included an error msg here. Will be pushed in the next commit.